### PR TITLE
Adding a recipe for ksw

### PR DESCRIPTION
**Pull request by nh13**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ksw/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+about:
+  home: https://github.com/nh13/ksw
+  license: MIT
+  license_file: LICENSE
+  summary: 'Ksw: (interactive) smith-waterman in C'
+
+extra:
+  identifiers:
+    - biotools:ksw
```
Comment (nh13) [inline recipes/ksw/meta.yaml]: I am not sure if this is needed; I looked at the `primer3` recipe.

File: recipes/ksw/build.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# These are not used ksw's makefile
+#export CFLAGS="-I$PREFIX/include"
+#export LDFLAGS="-L$PREFIX/lib"
+#export CPATH=${PREFIX}/include
```
Comment (nh13) [inline recipes/ksw/build.sh]: I'd be happy to patch the makefile directly if these are required.

**Comment by corburn** [comment]
744b5d4 I think you need to merge/rebase onto the master branch

**Comment by nh13** [comment]
@corburn thanks, done!

File: recipes/ksw/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/nh13/ksw/archive/v{{ version }}.tar.gz
```
Comment (corburn) [inline recipes/ksw/meta.yaml]: The URL does not have a 'v' before the version.
```
source:
  url: https://github.com/nh13/ksw/archive/{{ version }}.tar.gz
  sha256: 120a04e464a8fdb87a379a093a506b51537384ac930744a8136d7948cfab7b5b
```

**Comment by kyleabeauchamp** [comment]
Looks like your tests are failing

**Comment by nh13** [comment]
Thats progress! I’ll test locally I guess since I just found this: https://bioconda.github.io/contribute-a-recipe.html

**Comment by nh13** [comment]
Passes locally with `circleci build`, here's hoping.

**Comment by nh13** [comment]
Tests are passing
It's done

**Pull request merged**